src/Bundle/test/app/AppKernel.php

```diff
@@ -16,28 +16,14 @@
 
 class AppKernel extends Kernel
 {
-    public function registerBundles()
+    public function registerBundles(): iterable
```

Comment:
@pamil I just use that part of new Symfony architecture to have a something easier with all environments on travis.